src/data/markdown/docs/02 javascript api/12 k6-ws.md

```diff
@@ -2,15 +2,8 @@
 title: "k6/ws"
 excerpt: "k6 WebSocket API"
 ---
+<WsBlockquote />
```

Comment:
I guess there is no way to tell gatsby - "add this to the top for each page in this folder?" But instead we need to add it to everyone?